mobile/src/screens/meditation-home/components/add-meditation-modal/libs/constants.ts

```diff
@@ -0,0 +1,3 @@
+const EMPTY_ARRAY_LENGTH = 0;
```

Comment:
We have this common constant in shared folder, import it from there

Reply:
Yes, I've noticed that, but we've always created a file with constants in mobile. 

@Liza-Veis , @fshabanov , @v0ldemar01 , Do you think I should change it in this PR everywhere to use the constant from the shared folder?

Reply:
yes, you can change